src/analysis_and_optimization/Dependence_analysis.ml

```diff
@@ -231,7 +234,22 @@ let log_prob_build_dep_info_map (mir : Program.Typed.t) :
     Stmt.Fixed.{meta= Location_span.empty; pattern= SList mir.log_prob} in
   build_dep_info_map mir log_prob_stmt
 
+let reverse_mode_log_prob_build_dep_info_map (mir : Program.Typed.t) :
```

Comment:
The two new functions here both appear to be dead-code-on-arrival. The version for the normal log prob may also be, since it looks like it is only used in a test.